### PR TITLE
[docs] Fix workaround for functions worker errors

### PR DESCRIPTION
**Pull request by Jennifer88huang**
### Motivation
Users fail to restart broker with the current workaround.

### Modifications
Fix the workaround.

### CODE REVIEW

File: site2/docs/functions-worker.md [hunk 1]
```diff
@@ -270,16 +270,20 @@ The error message prompts when either of the cases occurs:
...
 
 ```bash
 bin/pulsar-admin namespaces get-clusters public/functions
 ```
 
-2. Check if the cluster is in the clusters list. If the cluster is not in the list, add it to the list and update the clusters list.
+3. Check if the cluster is in the clusters list. If the cluster is not in the list, add it to the list and update the clusters list.
 
 ```bash
 bin/pulsar-admin namespaces set-clusters --cluster=<existing-clusters>,<new-cluster> public/functions
```
Comment (tuteng) [inline site2/docs/functions-worker.md:284]: The right command `bin/pulsar-admin namespaces set-clusters --clusters <existing-clusters>,<new-cluster> public/functions`

Comment (Jennifer88huang) [reply site2/docs/functions-worker.md]: Thank you @tuteng, updated.

**Comment by tuteng** [comment]
Do other versions need to be fixed, e.g. 2.6.0, 2.6.1?

**Comment by Jennifer88huang** [comment]
Yes, sure, if it applies to other versions. I think we should update ever since 2.4.0.
We can ensure the update is correct, and then apply to other versions in another PR.

**Review by sijie — APPROVED** [review]

**Pull request merged**